.github/workflows/dotnet-desktop.yml

```diff
@@ -0,0 +1,59 @@
+name: .NET Core Desktop
```

Comment:
Eh, I guess it's not .NET Core. I cribbed this from elsewhere. The name doesn't super matter, but I can fix it...